content/sensu-go/5.15/reference/etcdreplicators.md

```diff
@@ -94,6 +97,27 @@ example      | {{< highlight shell >}}name: my_replicator{{< /highlight >}}
 
 #### Spec attributes
 
+ca_cert      |      |
+-------------|------
+description  | Path to an the PEM-format CA certificate to use for TLS client authentication.
+required     | true
+type         | String
+example      | {{< highlight shell >}}ca_cert: /path/to/trusted-certificate-authorities.pem{{< /highlight >}}
+
+cert         |      |
+-------------|------
+description  | Path to the PEM-format certificate to use for TLS client authentication.
+required     | true
+type         | String
+example      | {{< highlight shell >}}cert: /path/to/ssl/cert.pem{{< /highlight >}}
+
+key          |      |
+-------------|------
+description  | Path to the PEM-format key file associated with the `cert` to use for TLS client authentication.
+required     | true
+type         | String
+example      | {{< highlight shell >}}key: /path/to/ssl/key.pem{{< /highlight >}}
```

Comment:
These fields are only required if `insecure = false` which is the default configuration. However, if `insecure = true` these fields are not required.